.github/workflows/aws-marketplace-modeler.yaml

```diff
@@ -0,0 +1,63 @@
+name: "AWS Marketplace Mirror: Web Modeler"
```

Comment:
Cannot we simply make all components workflows as 1 workflow template? They are the same anyway :thinking:

Reply:
the aws_marketplace_ecr_mirror github action was intended to reduce the logic in each individual workflow. many of the components don't seem to use this action though.

Running it all as part of one action might be annoying if we just want to mirror one image, and then a script tries to download and upload all of the components.  But we could probably make a github action equivalent of the [mirror.sh](https://github.com/camunda/camunda-distribution/blob/4bf07dbb2a3330020badfb3af87d339cf204f9ba/aws-marketplace/mirror.sh) script which checks if an image exists before attempting to push it.

I'm not sure it makes much sense to refactor these sorts of things given we may be using a product to handle the mirroring for us in the near future.

Reply:
I'll do some small refactors. Not a huge deal. I added this commit:
https://github.com/camunda/camunda-distribution/pull/8/commits/1c2bcae73a3d1852e77e386920945b5ddc43302a

Though more could be done to further reduce the number of workflows.